app/build.gradle

```diff
@@ -35,16 +36,22 @@ android {
             testCoverageEnabled true
         }
     }
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+    buildToolsVersion '28.0.3'
```

Comment:
Should we restrict the `buildToolsVersion`? I think it's generally a good practice to keep it dynamic so that the latest default is used.
https://developer.android.com/studio/releases/build-tools

Reply:
removed, it will not take the latest version automatically.